core/Db/Adapter.php

```diff
@@ -119,4 +119,18 @@ public static function getAdapters()
 
         return $adapters;
     }
+
+    /**
+     * Checks if the available adapters are recommended by Piwik or not.
+     * @param string $adapterName
+     * @return bool
+     */
+    public static function isRecommendedAdapter($adapterName)
+    {
+        static $recommendedAdapters = array(
+            // Adapters recommended by Piwik. Needs to be in uppercase for string checking using in_array()
```

Comment:
just to have a bit simpler you could use `return $adapterName === 'PDO/MYSQL';` or even better `return strtolower($adapterName) === 'pdo/mysql';`

The `static` is not needed I think and it could simple be replaced with the one line above. We can refactor as soon as there is more than one recommended adapter